### ISSUE TITLE
Create MavlinkMessage Registerer

### ISSUE THREAD

**Issue by cb0s** [wuespace/telestion-extension-mavlink#4]
To manually register all MavlinkMessages and their IDs in the MessageIndex is quite a painful job. This is why I would like to add an automatic registerer. Any ideas how this could be implemented in a good way without mentioning all messages?

**Comment by fussel178** [comment]
Well, we have a `MavlinkMessageRegistrar` verticle to do that by give the package path to the mavlink message type.
Please see #37 for more information.

But the automatic registration still remains. :thinking:

**Comment by fussel178** [comment]
There are some ways in Java to call a function before the main application starts.
For example, static initializers.
(https://stackoverflow.com/questions/17337744/how-to-execute-a-particular-function-before-main-in-c-and-java)

It can register itself to the `MessageIndex`, so no more registrar?

What do you think? @cb0s

**Comment by cb0s** [comment]
Yeah this looks promising. I will have a look at this next weekend.
